build/charts/flow-aggregator/values.yaml

```diff
@@ -45,8 +45,18 @@ flowCollector:
 clickHouse:
   # -- Determine whether to enable exporting flow records to ClickHouse.
   enable: false
-  # -- DatabaseURL is the url to the database. TCP protocol is required.
+  # -- DatabaseURL is the url to the database. Provide the database URL as string with format
+  # <Protocol>://<ClickHouse server FQDN or IP>:<ClickHouse port>. The protocol has to be one
+  # from below: "tcp", "tls", "http", "https". When "tls" or "https" is used, tls will be enabled.
   databaseURL: "tcp://clickhouse-clickhouse.flow-visibility.svc:9000"
```

Comment:
According from the documentation https://github.com/ClickHouse/clickhouse-go, http port is usually 8123, and tls port is usually 9440. We probably want to mention it here? 
So far I don't see 9440 port used in CH [server](https://github.com/antrea-io/theia/blob/31f4752ce43b8a83f3d2ae5d31c0fb5b3941bc4c/build/charts/theia/README.md?plain=1#L40C1-L41), @yanjunz97 to your knowledge, when the TLS connection is directly between CH client and CH server, will server listen on 9440?

Reply:
After adding tls, there will be 4 ports for 4 protocols. Not sure if want to mention these 4 ports as it depends on user's setting?
Here is the  default protocol setting: https://clickhouse.com/docs/en/guides/sre/network-ports

Reply:
If the user would like to use CH server for TLS connection, they can set their customized port number for different protocols. I have a PR for this at https://github.com/antrea-io/theia/pull/348, which has not been merged yet.

And I think for the more common usage with Loadbalancer or Ingress, this port will also be user-defined. Thus I think it might be fine without mentioning a specific number like 9440. Maybe just remind the users of the port should be the same as the port they use for the selected protocol.

Reply:
Make sense to me. Thanks. It's fine to ignore this comment.